rclpy/src/rclpy_common/src/exceptions.cpp

```diff
@@ -33,4 +40,39 @@ RCLError::RCLError(const std::string & error_text)
 : std::runtime_error(append_rcl_error(error_text))
 {
 }
+
+void
+throw_if_unparsed_ros_args(py::list pyargs, const rcl_arguments_t & rcl_args)
+{
+  int unparsed_ros_args_count = rcl_arguments_get_count_unparsed_ros(&rcl_args);
+
+  if (unparsed_ros_args_count < 0) {
+    throw std::runtime_error("failed to count unparsed arguments");
+  } else if (0 == unparsed_ros_args_count) {
```

Comment:
@sloretz nit^N: that `else` is redundant, the previous `if` clause will leave the scope if it ever evaluates to `true`.

Reply:
removed else in ec35c464269d2883d1233c1a308b94323459d300